infra/utils.py

```diff
@@ -41,22 +42,28 @@ def execute(command, location=None, check_result=False):
     check_result: Should an exception be thrown on failed command.
 
   Returns:
-    The stdout of the command, the error code.
+    stdout, stderr, error code.
 
   Raises:
     RuntimeError: running a command resulted in an error.
   """
 
   if not location:
     location = os.getcwd()
-  process = subprocess.Popen(command, stdout=subprocess.PIPE, cwd=location)
+  process = subprocess.Popen(command,
+                             stdout=subprocess.PIPE,
+                             stderr=subprocess.PIPE,
```

Comment:
Are you sure we shouldn't just combine the stdout and stderr. Do we ever need them sepereately?
This would let us simplify the function by only having two return values.

Reply:
@Dor1s How do you feel about this? We had a discussion offline about making this function generic.

Reply:
Up to you guys. My original concern was that we were not returning stderr at all, it was just printed to the terminal automatically and lost.

I'm personally not a big fan of combining stdout and stderr together, but we do it in ClusterFuzz and some other places. Should be fine, and we can always separate it if we need it.

Reply:
I would rather not merge stdout and stderr for debugging reasons.  @jonathanmetzman but I could get rid of the stderr argument all together.

Reply:
We can leave it for now. But frankly I can't think of a single case where it would have been helpful not to combine them.